Add some warnings about the use of the Tombstone API (#6862)
pingcap/docs: scale-tidb-using-tiup.md

        
3. Wait for the store of the TiFlash node to disappear or for the `state_name` to become `Tombstone` before you stop the TiFlash process.

    If, after waiting for a long time, the node still fails to disappear or the `state_name` fails to become `Tombstone`, consider using the following command to force the node out of the cluster (use with caution).

    > **Warning:**
    > 

>     > - The following command directly discards the replicas on the TiFlash node, which might cause the query to fail.
>     > - You can only use the following command to the TiFlash node. Do not use this command to force any TiKV node out of the cluster. Otherwise, it will cause data loss.

Author: ```suggestion
    > - The following command directly discards the replicas on the TiFlash node, which might cause the query to fail.
    > - Once used this command, do not use PD Control to delete the nodes from the meta-information of PD. Otherwise, the information of Regions in the cluster will be inconsistent.
    > - You can only use the following command to the TiFlash node. Do not use this command to force any TiKV node out of the cluster. Otherwise, it will cause data loss.
```

        
3. Wait for the store of the TiFlash node to disappear or for the `state_name` to become `Tombstone` before you stop the TiFlash process.

    If, after waiting for a long time, the node still fails to disappear or the `state_name` fails to become `Tombstone`, consider using the following command to force the node out of the cluster (use with caution).

    > **Warning:**
    > 
    > - The following command directly discards the replicas on the TiFlash node, which might cause the query to fail.
    > - Once used this command, do not use PD Control to delete the nodes from the meta-information of PD. Otherwise, the information of Regions in the cluster will be inconsistent.

>     > - You can only use the following command to the TiFlash node. Do not use this command to force any TiKV node out of the cluster. Otherwise, it will cause data loss.

Reviewer: ```suggestion
    > - The following command is only applicable to TiFlash nodes. Do not use this command to force any TiKV node out of the cluster. Otherwise, data loss occurs.
```

>     > - After using this command, since the information of the replicas on the forced-out nodes still exists in the member information of other replicas, PD triggers scheduling to repair and clear these replicas' information of the forced-out nodes. Therefore, before using this command, you need to consider whether there are enough replicas to elect a leader after losing these nodes. Otherwise, PD cannot trigger scheduling to repair and clear the information, which will cause the service to be unavailable.

Reviewer: ```suggestion
    > - After using this command, because the information of the replicas on the forced-out node still exists in the member information of other replicas, PD triggers scheduling to repair and clear the replica information of the forced-out node. Therefore, before using this command, you need to consider whether there are enough replicas to elect a leader after losing the nodes. Otherwise, PD cannot trigger scheduling to repair and clear the information, which causes the service to be unavailable.
```
